cmd/root.go

```diff
@@ -91,7 +91,7 @@ func rootCmd(execs execs) (*cobra.Command, error) {
 			// so we cannot use a single viper structure, as described above.
 
 			// set up database connection
-			var dbconn database.Connection
+			//var dbconn database.Connection
```

Comment:
If this is extra and causes the other to be unused, this should be removed, not commented out.

Reply:
I pushed a new commit and this line of comments has been removed.